### PR TITLE
Create Agent for timer state

### PR DESCRIPTION
**Pull request by SimonLab**
Use agent to manage the state of the Timer

### CODE REVIEW

File: lib/stopwatch/application.ex [hunk 1]
```diff
@@ -13,9 +13,10 @@ defmodule Stopwatch.Application do
       # Start the PubSub system
       {Phoenix.PubSub, name: Stopwatch.PubSub},
       # Start the Endpoint (http/https)
-      StopwatchWeb.Endpoint
+      StopwatchWeb.Endpoint,
```
Comment (SimonLab) [inline lib/stopwatch/application.ex:16]: The application will create the timer by calling `Stopwatch.Timer.start_link\1` function

File: lib/stopwatch/timer.ex [hunk 2]
```diff
@@ -0,0 +1,15 @@
+defmodule Stopwatch.Timer do
+  use Agent
+
+  def start_link(_opts) do
+    Agent.start_link(fn -> ~T[00:00:00] end, name: __MODULE__)
```
Comment (SimonLab) [inline lib/stopwatch/timer.ex:5]: Module name are atoms so we can use it as the name for the timer agent

**Comment by SimonLab** [comment]
The stopwatch state is now shared between client using Agent. This can be tested at https://liveview-stopwatch.fly.dev/
I'm going to update the Readme to explain how it's done

**Review by nelsonic — APPROVED** [review]
@SimonLab superb addition! 🎉 

liveview-stopwatch-sync

Thanks! ⭐

**Pull request merged**